### ISSUE TITLE
Can't edit news image anymore.

### ISSUE THREAD

**Issue by buddh4** [nadilhassan/newsmodule#1]
When I try to edit an existing news entry, there is no way to change the image anymore, I don't see the image upload part. Only If I created the news without an image and want to edit it the image upload part is present.
![missingimageupload](https://cloud.githubusercontent.com/assets/9800854/18309830/8e7af454-74fb-11e6-88f9-a5b3d3b605f6.png)

**Comment by nadilhassan** [comment]
Yes, you are not able to edit, It's a limitation in this version, Hope we would integrate this feature on next release!

**Comment by buddh4** [comment]
ah ok ;) perhaps you should mark this post with an enhancement label or something.

**Comment by nadilhassan** [comment]
OH Sorry

**Comment by nadilhassan** [comment]
Yes, This is an enhancement.

**Comment by buddh4** [comment]
Hi nadil, just curious, is there a reason why the image can't be edited? Perhaps I can help you with this issue. I personally would expect beeing able to edit the content and image ;)

**Comment by nadilhassan** [comment]
The reason is there is no issue with editing the image but after editing the image If I save then the JSON output is printing on browser.So that why I removed the image editing option. OK . If you can tell me what way that to be done I hope I can do that
Thank You Very Much

**Comment by buddh4** [comment]
could you commit the version with your json problem and I'll have a look if this is not to much effort?

**Comment by nadilhassan** [comment]
HI! I am sorry for the late response, mess up with lot works.
I have created a gist of the files which causes the problem in editing image.
URL : https://gist.github.com/nadilhassan/49e11da9b920fbb7f3817b2c22fc2374
Please bear with the code structure! Sorry

**Comment by nadilhassan** [comment]
HI 
I have done updated the image editing feature
Can you please check this

**Comment by buddh4** [comment]
Hi, thanks will have a look soon!

**Comment by buddh4** [comment]
Hi Nadil, sorry did not forget this, but we have no time till the v1.2 release. I would suggest we'll release this after the v1.2, since the marketplace will open anyways. And I think the mdoule will need some alignments for v1.2.

**Comment by nadilhassan** [comment]
HI 
It's OK.. I can see that you guys are working hard. Great! No worries will publish it after the v1.2.  BTW what are the alignments needed?

**Comment by buddh4** [comment]
Perhaps in the stream I can help you with this, we'll also write a migration guide for module migrations.

**Comment by 001101** [comment]
Guides would be great!!
